test/integration/sys-mac-auth.int.c

```diff
@@ -0,0 +1,431 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ * Copyright (c) 2019, Intel Corporation
+ */
```

Comment:
Why this test? The original test this was copied from `sys-hmac-auth.int` which is something different. What we'd be looking to copy from is something that excerices `Tss2_Sys_HMAC*` commands. I don't see any tests for that. So it would follow the test you'd write for ESAPI for testing functionality.

1. Create an HMAC key since that's all thats supported right now.
2. HMAC it
3. Verify the HMAC.

It would look like `esys-hmac.int.c` just using the system API and the MAC interface. Same for an ESAPI version of the test.